smpplib/client.py

```diff
@@ -234,8 +234,10 @@ def accept(self, obj):
 
     def _message_received(self, p):
         """Handler for received message event"""
-        self.message_received_handler(pdu=p)
-        dsmr = smpp.make_pdu('deliver_sm_resp', client=self)
+        status = self.message_received_handler(pdu=p)
+        if status == None:
```

Comment:
that should be `status is None`

Reply:
Well.. there you have proof I am not a python programmer :) 
It's on the way...

Reply:
I force pushed my amended commit. Does it make it through to the PR?